packages/browserslist-config-cozy/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "browserslist-config-cozy",
+  "version": "0.1.0",
```

Comment:
1.0.0 ? It should already be stable enough?

Reply:
I did this since lerna will increase the version to publish and should publish 1.0.0 after this PR merged